requirements.txt

```diff
@@ -29,3 +29,4 @@ charset-normalizer
 dill >= 0.3.4
 pyOpenSSL >= 22.0.0
 pydateinfer==0.3.0
+pyarrow >= 9.0.0, < 10.0.0
```

Comment:
Where is pyarrow used?

Reply:
pandas depends on pyarrow to load parquet data. It doesn't come by default from pandas installation. It should be added explicitly. Refer: https://stackoverflow.com/questions/50760351/how-to-identify-pandas-backend-for-parquet

Reply:
Thanks